articles/flutter-vs-reactnative/index.md

```diff
@@ -0,0 +1,98 @@
+
+#### What React gets right
+##### Oh yes, JS!
+JS is one of the most popular languages among developers. That makes the transition for a web developer into mobile app development smoother.
+
+##### One codebase for mobile, web, and desktop apps
+JS facilitates developers to share code with web apps by creating abstraction components that can compile to target platforms. The world is your oyster in React! For example, Skype developed ReactXp to have a largely single codebase for Android, iOS, web and windows 10 UWP apps.
+
+##### More mature and a community far and wide
+No amount of money and resources can substitute maturity, and React is the most mature framework that is out there. You’ll find it difficult to encounter a question that has not already been answered in Stack Overflow. Anything you need to make your application over the top and rest assured you’ll find a 3rd party library for it.
+
+#### React vs Flutter, fight!
+##### Performance
+Nothing comes close to pure native apps in terms of performance. React Native and Flutter is no exception. But when we compare the two though, Flutter comes on top, because of how it achieves cross-platform. React Native has to use a JS bridge to communicate with native APIs, hence slowing things down. Flutter on the other hand compiles C/C++ library to Dart, since it's closer to metal and it can be consumed faster.
+
+##### Components out of the box
+Many of the common components that you’d need as a developer are not supplied in React Native. You’ll have to hunt down 3rd party libraries to achieve the desired task. Flutter on the other hand, supports a whole ranger of widgets (components). Material design and Cupertino design (Apple’s design philosophy) are supported. There are over 375 official widgets out of the box, and many more being added all the time.
+
+##### Software architecture and state management
+React native has a mature ecosystem around this. The developers have the freedom to choose which navigation package and global state management package they want without them worrying about how widely adopted it is.
+
+Flutter has only a handful of such packages, with each package not having widespread knowledge. For example, I use a package called stacked for state management, but it was hardly known in the community until now. This problem will surely go away in the future, with the rising adoption of Flutter as a framework.
+
+##### App sizes
+One of the bigger tradeoffs with the cross-platform is the app size, it is bound to be larger than native apps. But in this respect, it is also usually seen that React apps are relatively smaller than Flutter apps.
```

Comment:
This sentence could be a bit clearer or easier to read.

Reply:
I have tried to clear it out.